packages/pyright-internal/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
-        "outDir": "./out"
+        "outDir": "./out",
+        "paths": {
```

Comment:
Is this change needed?

Reply:
Yeah otherwise tsconfigResolveAliases throws an exception.